bundles/binding/org.openhab.binding.powermax/src/main/java/org/openhab/binding/powermax/internal/connector/PowerMaxConnector.java

```diff
@@ -0,0 +1,213 @@
+/**
+ * Copyright (c) 2010-2016, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.powermax.internal.connector;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.io.IOUtils;
+import org.openhab.binding.powermax.internal.message.PowerMaxBaseMessage;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * An abstract class for the communication with the Visonic alarm panel that
+ * handles stuff common to all communication types
+ *
+ * @author lolodomo
+ * @since 1.9.0
+ */
+
```

Comment:
normally there is no blank line between class comment and beginning of class definition.